unity-renderer/Assets/UnityGLTF/Scripts/NoThrottlingCounter.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace UnityGLTF.Scripts
+{
+    public class NoThrottlingCounter : IThrottlingCounter
```

Comment:
I prefer we adapt the pipeline to allow a null IThrottlingCounter in GLTFSceneImporter or otherwise add a public bool evaluateTimeBudget = true; in GLTFThrottlingCounter and use that inside EvaluateTimeBudge() as

new GLTFThrottlingCounter()
{
    evaluateTimeBudget = false
};

Instead of having this new `NoThrottlingCounter.cs`, it's like "the thing that should not be" from Metallica's cosmic horror song XD

Reply:
Deleted this class as it wasn't being used, If we encounter a throttle that should be disabled for other platform we can use the `enabled` field